chia/full_node/coin_store.py

```diff
@@ -394,6 +411,7 @@ async def rollback_to_block(self, block_index: int) -> List[CoinRecord]:
 
     # Store CoinRecord in DB and ram cache
     async def _add_coin_records(self, records: List[CoinRecord]) -> None:
+        txId = time_ns()
```

Comment:
I think I would prefer a counter here instead of asking for the time. We could just add a member to the class that we increment by one each time

Reply:
I was actually thinkin about some counter but didn't know what to use (Python practice), please suggest something you would use instead. I concluded getting a new tx-id should not be blocking or take a long time to retrieve.